plugins/chat/app/controllers/api_controller.rb

```diff
@@ -4,10 +4,68 @@ class Chat::Api < Chat::ChatBaseController
   before_action :ensure_logged_in
   before_action :ensure_can_chat
 
+  def result
+    @_result
+  end
+
```

Comment:
Yes, I’ve borrowed it from trailblazer :sweat_smile: 
The `Chat::Endpoint` class will set automatically the result object from the service in this instance variable and we make it accessible through this getter.

Reply:
Actually that changed a bit with my latest changes. Now that’s the `run_service` helper in `Chat::Api` that is responsible for this. It will be easier to maintain/change/whatever.